local-cli/runAndroid/runAndroid.js

```diff
@@ -71,10 +71,30 @@ function checkAndroid(args) {
   return fs.existsSync(path.join(args.root, 'android/gradlew'));
 }
 
+function getAdbPath() {
```

Comment:
Do we use `_` for private helper functions in this file? Should this be `_getAdbPath`?